### ISSUE TITLE
Question about model performance of the learning to count model

### ISSUE THREAD

**Issue by zipengxuc** [Cyanogenoid/vqa-counting#1]
Hi, I used the bottom-up attention features with 'output size = 36' and extracted text features by implementing 'python preprocessing_text.py'. I trained the full model on VQA v2 train-set and tested it on validation set. But the best accuracy I got was 64.58%, not as good as the performance you mentioned in the paper, the accuracy of which is around 65.42%. 
So could you please give me some suggestions on how to gain a better accuracy? I would truly appreciate it.

**Comment by Cyanogenoid** [comment]
I don't remember exactly how much the difference in performance between using 36 and 100 proposals is, but using 100 will definitely give you better results. Try training with the adaptive bottom-up features (remember to set output size to 100 again) and you should be able to get the results in the paper.

**Comment by zipengxuc** [comment]
Great thanks to you! I'll try it. :)

**Comment by Cyanogenoid** [comment]
Oops, I just realised that I didn't commit the download script to download the correct files, which is why this issue came up. It's now where it should be. (https://github.com/Cyanogenoid/vqa-counting/blob/master/vqa-v2/data/download.sh)
